module/zfs/dmu_zfetch.c

```diff
@@ -138,6 +145,7 @@ zfetch_fini(void)
 	wmsum_fini(&zfetch_sums.zfetchstat_misses);
 	wmsum_fini(&zfetch_sums.zfetchstat_max_streams);
 	wmsum_fini(&zfetch_sums.zfetchstat_io_issued);
+	aggsum_fini(&zfetch_sums.zfetchstat_io_active);
```

Comment:
Adding an ASSERT that the `zfetchstat_io_active` is zero before destroying it would be a nice way to verify the accounting is solid (and stays that way).

Reply:
I visually checked that it returns to zero in idle, but sure, I can add an assert for it.

Reply:
The CI does unload the modules after a full test run, so that's at least something.